src/ui/screens/settings/appPermissions/ViewAppPermissions.qml

```diff
@@ -64,6 +83,32 @@ MZViewBase {
         }
     }
 
+    Loader {
+        id: helpSheetLoader
+
+        active: false
+
+        onActiveChanged: if (active) item.open()
+
+        sourceComponent: MZHelpSheet {
+            title: MZI18n.HelpSheetsExcludedAppsTitle
+
+            model: [
+                {type: MZHelpSheet.BlockType.Title, text: MZI18n.HelpSheetsExcludedAppsHeader},
```

Comment:
nit: magic numbers

Reply:
will fix for all help sheets in a fast follow